### PR TITLE
[en] consistency: jack Jack JACK

### PR DESCRIPTION
**Pull request by trebmuh**
JACK is an acronym, a recursive one, which mean JACK Audio Connection Kit. Hence is should always be in capital letters.

### CODE REVIEW

File: _posts/2020-11-24-Multiple-Audio-Interfaces.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Under Windows, the best option to run multiple interfaces through Jamulus at the
 **Steps:**
 1. [Install and configure JACK on Windows](https://jackaudio.org/faq/jack_on_windows.html) - Make sure you select your best audio interface in the configuration steps, also add `-r 48000 -p <desired frame buffer size>` to ensure JACK is outputting audio in the required sample rate for Jamulus (48.000Hz). If your frame buffer size is too low for your hardware, you'll need to change it to a higher value;
 2. Install and open Voicemeeter;
-3. Open configured Jack PortAudio shortcut, Jack Control and Jamulus software;
+3. Open configured JACK PortAudio shortcut, Jack Control and Jamulus software;
```
Comment (ann0see) [inline _posts/2020-11-24-Multiple-Audio-Interfaces.md:22]: So what about Jack Control? Is this not all caps?

File: _posts/2020-11-24-Multiple-Audio-Interfaces.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ Under Windows, the best option to run multiple interfaces through Jamulus at the
...
 4. On Voicemeeter, go to HARDWARE OUT and on A1 dropdown list, select JackRouter. On Hardware Input 1 select your computer microphone. You can make your computer microphone have the lowest latency possible by going into _Menu > System Settings/Options..._ and enabling _WDM Input Exclusive Mode_ (if you experience crackling noises while speaking, disable this) and changing the _Engine Mode_ to _Swift_;
 5. Make sure the Hardware Input where you selected your microphone is sending sound through A1 by enabling the corresponding button next to the fader;
 6. On Jamulus, under Settings, select JackRouter as your Soundcard Device;
```
Comment (ann0see) [inline _posts/2020-11-24-Multiple-Audio-Interfaces.md:25]: To discuss: Soundcard, Sound card, Interface etc. There's already an issue about that.

**Review by ann0see — CHANGES_REQUESTED** [review]
Looks good. See my comments

File: _posts/2020-11-24-Multiple-Audio-Interfaces.md (hunk 1 above)
Comment (trebmuh) [reply _posts/2020-11-24-Multiple-Audio-Interfaces.md:22]: As far as I know, this is the correct spelling (see https://jackaudio.org/faq/jack_on_windows.html ).

File: _posts/2020-11-24-Multiple-Audio-Interfaces.md (hunk 2 above)
Comment (trebmuh) [reply _posts/2020-11-24-Multiple-Audio-Interfaces.md:25]: Correct, this is another consistency PR, please @ann0see keep it on topic, this one is about "jack/Jack/JACK".

**Comment by trebmuh** [comment]
Thanks for your review @ann0see .
I'm merging this one.

**Pull request merged**